### DISCUSSION TITLE
Checking that one element is added to a list

### DISCUSSION THREAD

**Discussion by baptistemanson** [cypress-io/cypress#14193] [Questions and Help]
I wish to test a keyboard shortcut that adds an element to a list.
```js
context("add", () => {
    beforeEach(() => {
      cy.visit("http://localhost:3000");
    });
  
    it("creates an element", async () => {
      let nbElements = 0;
      // first we count the number of elements we have in the list
      cy.get(".list")
        .children()
        .each(function () {
          nbElements++;
        })
        .then(() => {
          // check if an element was created
          cy.get(".list")
            .type("{ctrl}i", { force: true })
            .children()
            .its("length")
            .should("eq", nbElements + 1);
        });
    });
  });
  
```

2 questions:
- is there any more elegant way to do this in Cypress?
- ` should("eq", nbElements + 1)` is not evaluated anymore when I add `it("does nothing", () => {})` in the context. Why?

**Comment by bahmutov** [comment]
Hi there

- more elegant way to check if the list has grown by one item in https://github.com/cypress-io/cypress-fiddle/blob/master/cypress/integration/add-list-item.md

<img width="876" alt="Screen Shot 2020-12-16 at 12 36 06 PM" src="https://user-images.githubusercontent.com/2212006/102385225-98bdfa00-3f9b-11eb-8fb8-7a5e9fe6cb66.png">

for the second part, I do not understand what you mean it stops working. Can you provide a video or a screenshot?
